### PR TITLE
Allow workers to receive more than 1 argument if needed

### PR DESCRIPTION
**Pull request by pablomarti**
It bypasses to the original function when the worker class is `Sidekiq::Batch::Callback`.

Related error: https://rollbar.com/Chime/BankAccountService/items/649/occurrences/87622000711/

### CODE REVIEW

File: lib/simplekiq/processor.rb [hunk 1]
```diff
@@ -3,10 +3,14 @@
 module Sidekiq
   class Processor
     def execute_job(worker, cloned_args)
```
Comment (dcluna) [inline lib/simplekiq/processor.rb]:
```suggestion
    def execute_job(worker, **cloned_args)
```

File: lib/simplekiq/processor.rb [hunk 2]
```diff
@@ -3,10 +3,14 @@
...
-      raise ArgumentError.new("wrong number of arguments (given #{cloned_args.size}, expected 1)") unless cloned_args.size == 1
-      params = cloned_args.first.deep_symbolize_all_keys
-      worker.perform(params)
+      raise ArgumentError.new("wrong number of arguments (given #{cloned_args.size}, expected at least 1)") if cloned_args.size.zero?
+
+      if cloned_args.size == 1
+        params = cloned_args.first.deep_symbolize_all_keys
+        worker.perform(params)
```
Comment (dcluna) [inline lib/simplekiq/processor.rb]:
```suggestion
        worker.perform(**cloned_args.deep_symbolize_all_keys)
```

File: lib/simplekiq/processor.rb (hunk 1 above)
Comment (fzf) [inline lib/simplekiq/processor.rb]: You can just add this guard `return worker.perform(*cloned_args) if cloned_args['class'] == 'Sidekiq::Batch::Callback'`

**Comment by pablomarti** [comment]
@fzf I made some changes, what do you think?

**Comment by fzf** [comment]
@pablomarti be sure to test this change thoroughly on qa

**Pull request merged**